### DISCUSSION TITLE
Webpack bundle time increase

### DISCUSSION THREAD

**Discussion by juanfmiro** [TexteaInc/json-viewer#121] [Q&A] [answered]
Hello, there!

I am trying to use this library and component in a React app for a small extension I'm developing.
The results are very nice, and I managed to integrate it quite seamlessly with my app by customizing the theme. Everything worked out of the box, which is also very nice.

However, one thing I have noticed, which is currently a blocker for me, is that bundling/compilation time in Webpack jumps from less than one second (averaging 600-1000ms) to more than 5s. My webpack config is quite simple (mainly ts-loader only working for files in src/, and then a couple loaders for CSS/SCSS).

Have you experienced this in the past, or do you have any hints or guidelines for using the library with webpack? 

(Of course, I'm willing to add examples, webpack config or a simple demo project to showcase this, if it helps).

Thanks a lot for your effort and work!

**Comment by rtritto** [comment]
Covert project and/or this project to use yarn Plug'n'Play (nodeLinker: pnp)?

**Reply by pionxzh** [reply]
PnP will improve the consumer's speed?

**Reply by rtritto** [reply]
Yes, _pnp_ file contains the tree of dependencies to install them with no duplicates.

Source:
- https://yarnpkg.com/features/pnp
- https://medium.com/inato/make-your-ci-faster-and-improve-developer-experience-upgrade-to-yarn-2-with-or-without-939524d9bef2

**Comment by pionxzh** [comment, marked as answer]
This should be the same issue #140 which has been solved already. We also have some other PRs trying to improve it further.
Let me know if you still experience the same issue.
